.gitignore

```diff
@@ -1,4 +1,5 @@
 target
+druid-derive-data/Cargo.lock
```

Comment:
Is it more idiomatic to put it here, or as a separate .gitignore in the subdir?

Reply:
I'm not sure? I tend to only ever use one `.gitignore`, at the repo root.